cli/command/container/completion.go

```diff
@@ -165,6 +216,25 @@ func completeLink(cli command.Cli) func(cmd *cobra.Command, args []string, toCom
 	}
 }
 
+// completeLogDriver implements shell completion for the `--log-driver` option  of `run` and `create`.
+func completeLogDriver(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+	drivers := make([]string, 0, len(logDriverOptions))
```

Comment:
Technically we could get the list of log-drivers from the `info` response as well (I'm slightly on the fence if doing so would be "too heavy" or still acceptable)

Reply:
done